shopinvader/__manifest__.py

```diff
@@ -24,6 +24,7 @@
         "base_suspend_security",
         "base_url",
         "base_vat",
+        "connector",
```

Comment:
@simahawk IMO we don't need this dependency. `component`  should be enough. If `BaseShopinvaderConnectorComponent` inherit from `base` the code should still works and the dependency is no more required.

Reply:
as I added the inherit on collection.base to me it's better to keep the direct dependency explicit.
Anyway, I wonder why the backed was not inheriting from collection.base since we have many components registered for collection shopinvader.backend. 
Do you know why?

Reply:
but `collection.base` is defined into component not into `connector` https://github.com/OCA/connector/blob/12.0/component/models/collection.py#L58.
Components registered on `shopinvader.backend` are components registered for the rest services defined into shopinvader. `base_rest` woks with a pseudo collection created from the name declared into the `RestController` https://github.com/shopinvader/odoo-shopinvader/blob/12.0/shopinvader/controllers/main.py#L17 https://github.com/OCA/rest-framework/blob/12.0/base_rest/controllers/main.py#L189

Reply:
err. sorry, I messed up: the required dependency is connector.backend, hence it comes from connector

Reply:
but I prefer to remove the dependency on connector :cry: we don't need it. Our shopincader.backend can inherit directly from "collection.base". There is no added value provided by connector at this stage.

Reply:
ok, I've tried it. If I inherit only from `collection.base` it fails like:
```
  File "/odoo/src/odoo/modules/registry.py", line 62, in __new__
    return cls.new(db_name)
  File "/odoo/src/odoo/modules/registry.py", line 86, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/odoo/src/odoo/modules/loading.py", line 417, in load_modules
    force, status, report, loaded_modules, update_module, models_to_check)
  File "/odoo/src/odoo/modules/loading.py", line 313, in load_marked_modules
    perform_checks=perform_checks, models_to_check=models_to_check
  File "/odoo/src/odoo/modules/loading.py", line 188, in load_module_graph
    model_names = registry.load(cr, package)
  File "/odoo/src/odoo/modules/registry.py", line 240, in load
    model = cls._build_model(self, cr)
  File "/odoo/src/odoo/models.py", line 467, in _build_model
    ModelClass.__bases__ = tuple(bases)
TypeError: Cannot create a consistent method resolution
order (MRO) for bases connector.backend, base, collection.base, BaseModel - - -
```
but this is really weird as those mixins are very basic. Any clue?

Reply:
@simahawk here it's https://github.com/camptocamp/odoo-shopinvader/pull/1